ios/AppDelegate.m

```diff
@@ -48,7 +49,11 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
   [[ExposureManager shared] setup];
 
   [RNSplashScreen showSplash:@"LaunchScreen" inRootView:rootView];
-  [Bugsnag start];
+  
+  // Read dev environment variable
+  if ([[ReactNativeConfig envFor:@"DEV"]  isEqual: @"true"]) {
+    [Bugsnag start];
```

Comment:
I think it will cause a runtime error if the library is referenced and we didn't start it.
That's why I used `releaseStages` instead https://github.com/Path-Check/gaen-mobile/pull/258 and it's what I found in their repository https://github.com/bugsnag/bugsnag-js/issues/135

Reply:
You're 💯 right @juancruzgs but we would have to mimic that logic on the javascript side also since we need to start bugsnag in all those three places, this felt easier for this round but we definitely need a way to share that config around